package.json

```diff
@@ -68,5 +69,8 @@
         "vite": "^5.1.2",
         "vite-plugin-dts": "^3.7.2",
         "vitest": "^1.2.2"
+    },
+    "dependencies": {
+        "ua-parser-js": "^1.0.37"
```

Comment:
I guess you need to move this to the devdependency 🤔 

cc: @yashim-deriv

Reply:
it will be used in the actual application code that runs in production though, shouldn't it put this to dependency? @niloofar-deriv

Reply:
Things in the library is a little bit different 🤔

Reply:
My bad @shontzu-deriv you don't need to change this
please ignore my comment 🙏